pontoon/base/templates/translate.html

```diff
@@ -166,6 +166,18 @@
                 <td><span>Ctrl</span>&nbsp;+&nbsp;<span>Alt</span>&nbsp;+&nbsp;<span>F</span></td>
               </tr>
               <tr>
+                <td>Suggest a placeable</td>
+                <td>
+                    <span>Ctrl</span>&nbsp;+&nbsp;<span>Alt</span>&nbsp;+&nbsp;<span>P</span><br>
+                </td>
+              </tr>
+              <tr>
+                <td>Select the next placeable</td>
+                <td>
+                  <span>Tab</span>
```

Comment:
Tab keyboard shortcut is already taken for navigation between helpers' suggestions.

We could we use Up + Down instead, like we do in menus.